test_packages.txt

```diff
@@ -13,3 +13,4 @@ onnx<=1.13.0; python_version < '3.10'
 onnx; python_version >= '3.10'
 onnxruntime<=1.11.0; python_version < '3.10'
 onnxruntime; python_version >= '3.10'
+scikit-learn>=1.1.0,<1.3.0
```

Comment:
Why do you need the upper constraint here?
Like I can understand `<2.0`, but why constraint on minor version?